node/src/chain_spec/turing.rs

```diff
@@ -100,7 +101,7 @@ pub fn turing_latest_latest() -> ChainSpec {
 		"turing",
 		ChainType::Live,
 		move || {
-			let allocation_json = &include_bytes!("../../../distribution/neumann_alloc.json")[..];
+			let allocation_json = &include_bytes!("../../../distribution/turing_alloc.json")[..];
```

Comment:
`pub fn turing_staging()` -> `pub fn turing_llive()

Let's do 2 functions for this one. The relay for staging can point to rococo and the live one can point to kusama

Reply:
i'm assuming we want this current allocation/vesting on the live one, right? for the staging one, do we want the same vesting? or different behaviors at all?